frontend/src/store/auth/index.js

```diff
@@ -4,6 +4,11 @@ import CV from './CV'
 
 import { createProfile, updateProfile, getSelf } from '@/utils/api/auth'
 
+// TODO: more idiomatic way to do this? Router something?
+const _baseURL = `${window.location.protocol}//${window.location.host}`
```

Comment:
base url should be exposed by as VUE_APP_URL

Reply:
hmmm - this won't work on localhost however - I'm not sure why but you removed `VUE_APP_URL=localhost:8080` from the .env file...

In any case the port may change (I use syncthing and it uses port 8080 by default :/)

So either this should stay or VUE_APP_URL should be fixed, but idk how to make it work for every environment

Reply:
i think imma leave it for now